ui/Screen/Onboarding.svelte

```diff
@@ -44,7 +44,14 @@
       await session.createKeystore(passphrase);
       // Retry until the API is up
       const identity = await withRetry(
-        () => createIdentity({ handle }),
+        async () => {
+          try {
+            return await createIdentity({ handle });
+          } catch (_) {
+            // The proxy might fail the first request after store creation
+            throw new Error("Failed to fetch");
+          }
+        },
```

Comment:
The `withRetry` only fails if it gets a `Failed to fetch` message, meaning the connection didn't work. Here, the proxy is returning a `403` and it's not retrying as it has a different error message.

I'm wrapping it to trigger the retry on 403 as well. There is likely a more elegant way to allow retries on different status code but I mostly wanted to get it working.

Reply:
This is curious: `createIdentity` should never return a 403 :thinking:. If it does, there is something wrong and we don’t want to retry.  Can you reproduce this @bltavares? If so, we should open a separate issue.

Reply:
It happens frequently, but sometimes it works. There seem to be a race condition on the proxy, where the `session.createKeystore()` respond before the session token is stored, and then `createIdentity` fails to authenticate. #1496 seems to showcase it's happening on other OSes

Reply:
To me it seems like #1496 describes an issue where identity creation fails permanently and requires manual interaction to resolve. If we keep retrying this would just delay showing an error. If it fails once and then recovers we also loose valuable information about existing errors if we keep re-trying. This is why I think we should not include this code and find a solution to the underlying issue.

Could you add error information to #1496. You can get it by clicking “copy to clipboard” when an error shows. Our error reporting infrastructure has improved since the issue was opened.

Reply:
I removed this in https://github.com/radicle-dev/radicle-upstream/pull/2375.